packages/api/src/models/Validator.js

```diff
@@ -1,12 +1,66 @@
 module.exports = class Validator {
   proTxHash
-
-  constructor (proTxHash) {
+  blocksCount
+  lastProposedBlockHeader
+  constructor (
+    proTxHash,
+    latestHeight,
```

Comment:
Look at the block example. I think arguments should be nested, rather than plain

```
const BlockHeader = require('./BlockHeader')

module.exports = class Block {
  header
  txs

  constructor (header, txs) {
    this.header = header ?? null
    this.txs = txs ?? null
  }

  static fromRow ({ header, txs }) {
    return new Block(BlockHeader.fromRow(header), txs)
  }
}

```

Reply:
✅